Sources/Deprecated.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Deprecated.swift
+//  APIKit
+//
+//  Created by Yosuke Ishikawa on 9/19/16.
+//  Copyright © 2016 Yosuke Ishikawa. All rights reserved.
+//
+
+import Result
+
+// MARK: - Protocols
+
+@available(*, deprecated, renamed: "Request")
```

Comment:
Right. 

As you said, we should not contain these APIs in final `3.0.0` release. Since there're no other tasks that must be finished until `3.0.0`, I'm going to replace them with `unavailable` on this branch.